MetaMask/metamask-extension
# Backport the `@metamask/message-manager@v5` update

## Explanation

The `@metamask/message-manager` update to v5 has been backported to the v10.30.x release branch. This update includes three breaking changes, but two (bump in minimum supported Node.js version, and change in type from `Map` to `Record`) don't affect this project. The only breaking change requiring changes was the addition of the `getCurrentChainId` constructor parameter for the TypedMessageManger.

See this PR for more information: https://github.com/MetaMask/metamask-extension/pull/19078

## Manual Testing Steps

See this PR for more information: https://github.com/MetaMask/metamask-extension/pull/19078

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
